ros2_batch_job/__main__.py

```diff
@@ -63,7 +63,7 @@
     'importlib-metadata',
     'lark-parser',
     'mock',
-    'mypy',
+    'mypy==0.761',
```

Comment:
Because mypy is included here we're actually using this pip package even after installing mypy via apt on Ubuntu. That's why this test was also failing on our Ubuntu platforms where it wouldn't be if we were truly using the system package.